### PR TITLE
Cloud 2002.0.14 release

### PR DESCRIPTION
**Pull request by hguthrie**
Updates to Cloud documentation for the 2002.0.14 release

- [x] link check
- [x] technical review
- [x] add whatsnew items

whatsnew
Updated the [Release notes for ece-tools version 2002.0.14.](
http://devdocs.magento.com/guides/v2.2/cloud/release-notes/cloud-tools.html#v2002014)
Renamed Local environment setup to Local development. Added new [Docker development](https://devdocs.magento.com/guides/v2.2/cloud/reference/docker-development.html) architecture.
Added [Reset cron jobs](http://devdocs.magento.com/guides/v2.1/cloud/trouble/reset-cron-jobs.html) topic to version 2.1. 
And other minor Cloud guide fixes for broken links and a bad redirect for the ece-tools update.

### CODE REVIEW

**Comment by hostep** [comment]
@hguthrie or @meker12, just FYI: it looks like there slipped in some merge conflict in the file `guides/v2.3/magento-functional-testing-framework/2.3/getting-started.md`:
```
<<<<<<< HEAD
=======
{% if page.guide_version == "2.2" %}
{% include_relative include/note-2.3-docs.md %}
{% endif %}
 >>>>>>> Removing access-acct files and adding setup folder
```

**Comment by hguthrie** [comment]
@hostep This Branch is going through some updates at the moment. Yes, we caught that. Thanks for checking on us! :)

**Comment by hguthrie** [comment]
Okay, so the corrections have been made to remove any conflicts. @meker12 Check that the release-related work is intact.

File: guides/v2.1/cloud/release-notes/cloud-tools.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ functional_areas:
   - Setup
   - Configuration
 ---
-
-<!-- Assigning liquid variables for placeholder values
+<!--MAGECLOUD
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/release-notes/cloud-tools.md]: What is the purpose of this block of liquid tags?

File: guides/v2.1/cloud/release-notes/cloud-tools.md [hunk 2]
```diff
@@ -25,17 +25,60 @@ php ./vendor/bin/ece-tools list
...
+
+-  **Cron-related fixes**
+
+   -  <!--MAGECLOUD-2602-->Fixed a cron scheduling issue that prevented jobs from running if you specify a cron frequency other than the default (1 minute).
+
+   -  <!--MAGECLOUD--2537-->Fixed an issue in the deploy phase that allowed cron jobs to continue running during deployment, which can cause database locks and other critical issues. Now, all cron jobs are stopped before the deploy phase begins and restarted after the deployment completes.
+   
+   -  <!--MAGECLOUD-2501-->Fixed the cron job workflow in versions 2.1.4 through 2.1.x to unlock frozen cron jobs so that they can be stopped before beginning deployment. Previously, a frozen cron job caused the deployment to stall.
+   
+-  <!--MAGECLOUD-2527-->Changed the format of the `config.php` file generated by the `vendor/bin/ece-tools config:dump` command to use short array syntax and 4-space indentation to comply with Magento coding standards./:q
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/release-notes/cloud-tools.md]: Looks like there are extra random characters at the end of this line?

>... coding standards./:q

File: guides/v2.1/cloud/setup/first-time-setup.md [hunk 3]
```diff
@@ -1,60 +1,54 @@
...
-* [Git](https://git-scm.com/){:target="_blank"} for repository and code management, [branching methodologies](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows){:target="_blank"}, and working with Magento Git code
-* [GitHub](https://github.com/){:target="_blank"} for managing your code and accessing Magento repos
-* [Fastly](https://www.fastly.com/){:target="_blank"} for CDN and caching, used in Staging and Production
-* [Galera Cluster](http://galeracluster.com/){:target="_blank"} for databases, used in Staging and Production (with triple redundancy failover in Production)
+-  **Magento Access Key**  
+    Provides secure access through a 32-character authentication token. The token authenticates between your account, the Magento 2 code repo (repo.magento.com), and any other Git services like GitHub and 3rd party accounts needed for development. You can create multiple Magento access keys. For the workspace setup, you can start with one specific key for your code repository. To generate a key, contact the Project Owner to generate [Magento authentication keys]({{ page.baseurl }}/install-gde/prereq/connect-auth.html).
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/setup/first-time-setup.md]: I think we should be clear that repo.magento.com is . Composer repo. For example:

> The token authenticates between your account, the Magento 2 Composer repo (repo.magento.com)...

File: guides/v2.1/cloud/setup/first-time-setup.md [hunk 4]
```diff
@@ -1,60 +1,54 @@
...
+-  **Cloud Project account**  
+    The Project Owner or Technical Admin (Super User) should invite you to the {{site.data.var.ece}} project. When you receive the e-mail invitation, click the link and follow the prompts to create your account. See [Set up an account]({{ page.baseurl }}/cloud/before/before-workspace.html#newaccount) for details.
+-  **Magento Encryption Key**  
+    When importing an existing Magento system only, capture the Magento encryption key used to protect your access and data for the Magento database. For details on this key, see [Resolve issues with encryption key]({{ page.baseurl }}/cloud/trouble/trouble-crypt-key-variable.html)
 
-## Gather credentials {#credentials}
+## Launch a Docker environment {#docker}
 
-Prior to setting up your workspace, you may want to gather the following credentials and accounts:
+You can use the Docker environment to emulates the {{site.data.var.ece}} Integration environment for local development. You need three, essential components: a {{site.data.var.ee}} v2 template, Docker Compose, and {{site.data.var.ece}} `{{site.data.var.ct}}` package.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/setup/first-time-setup.md]: Subject/verb agreement

> You can use the Docker environment to emulate**~s~**

File: guides/v2.1/cloud/release-notes/cloud-tools.md (hunk 2 above)
Comment (hguthrie) [reply guides/v2.1/cloud/release-notes/cloud-tools.md]: Maybe it's an emoji. 😯

File: guides/v2.1/cloud/release-notes/cloud-tools.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.1/cloud/release-notes/cloud-tools.md]: The tags and values look like circular references. I don't understand the utility of these. Why not just use plain text? The tags don't appear to offer any convenience. Their presence is confusing.

Comment (meker12) [reply guides/v2.1/cloud/release-notes/cloud-tools.md]: The intent was to show the actual values that cause the deployment error.  Those values are present in the error message. That's why I included them as is.

Comment (dshevtsov) [reply guides/v2.1/cloud/release-notes/cloud-tools.md]: @meker12 
I didn't recommend it, I shown how to use it.

**Comment by hguthrie** [comment]
@dshevtsov Just letting you know that we added the whatsnew. We fixed a lot of links. And we are simply waiting for Cloud to release and then we can merge. You are welcome to give it a check while we wait.

**Comment by dshevtsov** [comment]
@hguthrie the the __Docker development__ link in the whatsnew returns 404

**Comment by meker12** [comment]
Fixed the link in the comment.

**Pull request merged**